Microsoft.Packaging.Tools/docs/trimming.md

```diff
@@ -0,0 +1,73 @@
+#Dependency Trimming
+
+This package provides build infrastructure for trimming the output of an application.
+
+It determines what is used by the application by examining static dependencies of the application binary as well as any directly referenced packages.  For any file that is unused it will be removed from the set of files copied to the output and publish folders and removed from the application's dependency file(`deps.json`) in the case of a .NET Core application.
+
+## How to use
+First install the `Microsoft.Packaging.Tools` package in your application.
+
+You must use *Visual Studio 2017* or later, or *.NET Core command-line (CLI) tools 1.0* or later.
+
+### From the commandline
+Specify `/p:TrimUnusedDependencies=true` when building the project with either `dotnet` or `msbuild`.
+
+Examples:
+```
+dotnet build /p:TrimUnusedDependencies=true
+dotnet publish /p:TrimUnusedDependencies=true
+msbuild /p:TrimUnusedDependencies=true
+msbuild /t:Publish /p:TrimUnusedDependencies=true
+```
+
+### From the IDE or committing the change to your project
+
+In your project (*.csproj* file) make the following change.
+
+```xml
+<PropertyGroup>
+  <TrimUnusedDependencies>true</TrimUnusedDependencies>
+</PropertyGroup>
+```
+
+### Additional options
+`@(TrimFilesRootFiles)` -  Additional *root* files to consider part of the application.  See [roots](#roots).  
```

Comment:
I'd provide an example project that has used a couple of these options. People that don't know MSBuild will be a bit thrown off by some of the syntax.